ev3dev/core.py

```diff
@@ -182,8 +183,16 @@ def get_index(file):
                 self._device_index = get_index(name)
                 self.connected = True
             except StopIteration:
+                self._path = None
+                self._device_index = None
                 self.connected = False
 
+    def __str__(self):
```

Comment:
Say you do the following:
`motor = LargeMotor(OUTPUT_B)`
`log.info("%s: move to position %d" % (motor, args.degrees))`


Without the \__str__ code here you will see output such as:
`2016-02-16 02:06:12,195  INFO: <ev3dev.helper.LargeMotor object at 0xb63aa270>: move to position 45`

but with this \__str__ code you will see:
`2016-02-16 02:05:07,880  INFO: LargeMotor(outB): move to position 45`

It just makes it much easier to print/log an object because you can get something that is more human readable.